crypto/fipsmodule/CMakeLists.txt

```diff
@@ -320,13 +320,15 @@ if(FIPS_DELOCATE)
   # CMakeLists.txt that defines it. Thus we have to wrap bcm.o in a custom target
   # so that crypto can depend on it.
   add_custom_target(bcm_o_target DEPENDS bcm.o)
+  set(BCM_NAME bcm.o PARENT_SCOPE)
```

Comment:
Why is this setting needed? I understand this is in the scope of static build on Linux, right?

Reply:
`PARENT_SCOPE` means the CMake file in `crypto` directory which needs to know `BCM_NAME`. The same is done in line 445 for the shared build.